checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -500,100 +579,119 @@ def __init__(self, title, tests, rerun_candidates):
-            group, self._tp_info.get('name'), state=False)
+            group, self._tp_info.get("name"), state=False
+        )
 
     def label_toggle(self):
         if self.is_name:
-            self.set_label(self._tp_info.get('id'))
+            self.set_label(self._tp_info.get("id"))
             self.is_name = False
         else:
-            self.set_label(self._tp_info.get('name'))
+            self.set_label(self._tp_info.get("name"))
             self.is_name = True
 
     @property
     def tp_id(self):
-        return self._tp_info.get('id')
+        return self._tp_info.get("id")
 
     @property
     def name(self):
-        return self._tp_info.get('name')
-
+        return self._tp_info.get("name")
 
-class TestPlanBrowser():
 
+class TestPlanBrowser:
     palette = [
-        ('focus', 'black', 'light gray', 'standout'),
-        ('body', 'light gray', 'black', 'standout'),
-        ('head', 'black', 'light gray', 'bold'),
-        ('buttnf', 'black', 'light gray'),
-        ('buttn', 'light gray', 'black', 'bold'),
-        ('foot', 'light gray', 'black'),
-        ('start', 'dark green,bold', 'black'),
+        ("focus", "black", "light gray", "standout"),
+        ("body", "light gray", "black", "standout"),
+        ("head", "black", "light gray", "bold"),
+        ("buttnf", "black", "light gray"),
+        ("buttn", "light gray", "black", "bold"),
+        ("foot", "light gray", "black"),
+        ("start", "dark green,bold", "black"),
     ]
 
-    footer_text = [('Press '), ('start', '<Enter>'), (' to continue')]
-
-    help_text = urwid.ListBox(urwid.SimpleListWalker([
-        urwid.Text(('focus', " Keyboard Controls and Shortcuts "), 'center'),
-        urwid.Divider(),
-        urwid.Text("Select/Deselect                 Space"),
-        urwid.Text("Navigation                      Up/Down"),
-        urwid.Text("                                Home/End"),
-        urwid.Text("                                PageUp/PageDown"),
-        urwid.Text("Toggle test plan id/summary     i"),
-        urwid.Text("Filter test plan list           f,s,/"),
-        urwid.Text("Exit (abandon session)          Ctrl+C")]))
-
-    def __init__(self, title, test_plan_list, selection=None):
+    footer_text = [("Press "), ("start", "<Enter>"), (" to continue")]
+
+    help_text = urwid.ListBox(
+        urwid.SimpleListWalker(
+            [
+                urwid.Text(
+                    ("focus", " Keyboard Controls and Shortcuts "), "center"
+                ),
+                urwid.Divider(),
+                urwid.Text("Select/Deselect                 Space"),
+                urwid.Text("Navigation                      Up/Down"),
+                urwid.Text("                                Home/End"),
+                urwid.Text("                                PageUp/PageDown"),
+                urwid.Text("Toggle test plan id/summary     i"),
+                urwid.Text("Filter test plan list           f,s,/"),
```

Comment:
The UI for resuming the sessions is really cool!
Should we add the "R" command also to the help menu?
`                urwid.Text("Resume session (if available)       r"),`

Reply:
Absolutely, thanks!